src/main/java/edu/ie3/datamodel/utils/ContainerUtils.java

```diff
@@ -418,4 +418,236 @@ public static JointGridContainer combineToJointGrid(
     return new JointGridContainer(
         gridName, rawGrid, systemParticipants, graphicElements, subGridTopologyGraph);
   }
+
+  /**
+   * Modifies a given {@link SubGridContainer} to make it computable for power flow calculations
+   * based on its structure. This means, that the grid is modified in a way that slack nodes are
+   * added for transformers based on assumptions about the grid, as well as all other affect
+   * entities of the grid are accordingly.
```

Comment:
I would give a little more more insight on why this is even necessary. Some suggestion:

```
Returns a copy of the given {@link SubGridContainer} with a certain set of nodes marked as slack nodes. By default, when the container is derived from {@link JointGridContainer}, only the original slack nodes are incorporated in the different sub containers. Thereby, power flow calculations of most of them cannot be carried out right away. This means, that the grid is modified in a way that slack nodes are added at transformer nodes based on assumptions about the grid, as well as all other affect entities of the grid are accordingly.
```